notebooks/week2/02_04_train_log_custom_model.py

```
@@ -0,0 +1,169 @@
+# Databricks notebook source
+import mlflow
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)